org.rolisteam.rolisteam.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.rolisteam.rolisteam</id>
+  <name>Rolisteam RPG Client</name>
+  <developer_name>Rolisteam</developer_name>
+  <summary>Client Companion to Rolisteam Server</summary>
+  <description>
+    <p>This is the Rolisteam Client Binary which can be used either standalone as a server/client or connect to a Roliserver instance (currently a docker image)</p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv2</project_license>
+  <url type="homepage">https://www.rolisteam.org/</url>
```

Comment:
The appdata lacks release, launchable and  OARS tags.

Reply:
For the launchable and OARS, ok but the release It is ok at the end appdata.xml. Am I wrong ?